processor/k8sattributesprocessor/internal/kube/client.go

```diff
@@ -519,11 +519,11 @@ func (c *WatchClient) addOrUpdatePod(pod *api_v1.Pod) {
 
 	for _, id := range c.getIdentifiersFromAssoc(newPod) {
 		// compare initial scheduled timestamp for existing pod and new pod with same identifier
-		// and only replace old pod if scheduled time of new pod is newer or equal.
+		// and only replace old pod if scheduled time of new pod is newer.
```

Comment:
The old comment should still stand here: the pod object will get replaced if the new pod has a starting time equal to or greater than that of the old pod.

Reply:
thanks, good point!